src/ops/ops.go

```diff
@@ -181,6 +181,15 @@ func (o *ops) SetBootOrder(device string) error {
 		return nil
 	}
 
+	o.log.Infof("SetBootOrder, runtime.GOARCH: %s, device: %s", runtime.GOARCH, device)
```

Comment:
I'm pretty sure ``runtime.GOARCH`` applies to complie-time (despite its name containing ``runtime``)
I guess it's not too critical, but if it's easily possible to use real runtime information it's better

Reply:
Build time and run time CPU arch have to be the same, no code can build on one platform and run on other platform.